airbyte-cdk/python/unit_tests/sources/declarative/interpolation/test_jinja.py

```diff
@@ -47,6 +47,48 @@ def test_literals(test_name, s, value):
     assert val == value
 
 
+@pytest.mark.parametrize(
+    "test_name, context, input_string, expected_value",
+    [
+        ("test_get_value_from_stream_slice",
```

Comment:
Using the `pytest.param` class is more idiomatic for how we should be structuring our tests.
```
"context, input_string, expected_value"
[
  pytest.param(
     "input_1",
     { ... },
      id="test_get_value_from_stream_slice",
  )
]
```